docs/public/specifications/tests/Nameserver-TP/nameserver15.md

```diff
@@ -0,0 +1,119 @@
+# NAMESERVER15: Checking for software version
```

Comment:
Usually it says "Test for". Is "checking" better?

Reply:
How about naming this test “NAMESERVER15: Test for name servers revealing their software's version numbers”?